packages/react-native-client/src/components/VideoPreviewView.tsx

```diff
@@ -30,12 +12,15 @@ const NativeView: React.ComponentType<VideoPreviewViewProps> =
  *
  * @category Components
  * @param {object} props
- * @param {string} props.cameraId
  */
-export const VideoPreviewView = React.forwardRef<
-  React.ComponentType<VideoPreviewViewProps>,
-  VideoPreviewViewProps
->((props, ref) => (
-  // @ts-expect-error ref prop needs to be updated
-  <NativeView {...props} captureDeviceId={props.cameraId} ref={ref} />
-));
+export const VideoPreviewView = (props: VideoPreviewViewProps) => {
+  const { localPeer } = usePeers();
+
+  const cameraTrack = localPeer?.cameraTrack;
+
+  if (!cameraTrack) {
+    return null;
+  }
+
+  return <VideoRendererView trackId={cameraTrack.id} {...props} />;
```

Comment:
I would switch order, to always overwrite `trackId` param (in case someone will ignore TS) 
```suggestion
  return <VideoRendererView {...props} trackId={cameraTrack.id}  />;
```